src/core/services/managers/marker-manager.ts

```diff
@@ -73,6 +79,10 @@ export class MarkerManager {
       title: marker.title
     });
     this._markers.set(marker, markerPromise);
+
+    this._mapsWrapper.getAnimation(marker.animation).then((anim) => {
```

Comment:
please delete this an add the animation in the _mapsWrapper.createMarker method.